dask_cuda/benchmarks/local_cupy.py

```diff
@@ -1,30 +1,25 @@
-import asyncio
-from json import dumps
+import contextlib
+from collections import ChainMap
 from time import perf_counter as clock
-from warnings import filterwarnings
 
 import numpy as np
+import pandas as pd
 from nvtx import end_range, start_range
 
-import dask
 from dask import array as da
-from dask.distributed import Client, performance_report, wait
-from dask.utils import format_bytes, format_time, parse_bytes
+from dask.distributed import performance_report, wait
+from dask.utils import format_bytes, parse_bytes
 
+from dask_cuda.benchmarks.common import Config, execute_benchmark
 from dask_cuda.benchmarks.utils import (
-    get_cluster_options,
-    get_scheduler_workers,
-    hmean,
-    hstd,
     parse_benchmark_args,
-    peer_to_peer_bandwidths,
     print_key_value,
     print_separator,
-    setup_memory_pool,
+    print_throughput_bandwidth,
 )
 
 
-async def _run(client, args):
+def bench_once(client, args, write_profile=None):
```

Comment:
I made the clusters and clients `asynchronous=False`, all the futures are immediately `await`ed on, so it didn't seem necessary to have async stuff littering the code.